src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,149 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T value;
+        private Node<T> next;
+        private Node<T> previous;
+
+        public Node(T value, Node<T> next, Node<T> previous) {
```

Comment:
Better change sequence of inputs.What you think about previous,value,next?

Reply:
It's a good idea! Thanks